drivers/timer/riscv_machine_timer.c

```diff
@@ -14,7 +14,7 @@
 				 / (uint64_t)CONFIG_SYS_CLOCK_TICKS_PER_SEC))
 #define MAX_CYC INT_MAX
 #define MAX_TICKS ((MAX_CYC - CYC_PER_TICK) / CYC_PER_TICK)
-#define MIN_DELAY 1000
+#define MIN_DELAY 100
```

Comment:
I wonder if this shouldn't become a config option, then.

Reply:
Sure, good idea.